common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -189,7 +189,7 @@ object CometConf extends ShimCometConf {
           "'spark.shuffle.manager' must be set before starting the Spark application and " +
           "cannot be changed during the application.")
       .booleanConf
-      .createWithDefault(false)
+      .createWithDefault(true)
```

Comment:
We also need to fix default value in description

Reply:
We should probably automate adding the text stating the default value. Thanks for catching that.